web/package.json

```diff
@@ -5,12 +5,15 @@
   "author": "Daniel G. Taylor",
   "license": "MIT",
   "dependencies": {
-    "wintersmith-coffee": "~0.2.0",
-    "wintersmith-stylus": "~0.4.0",
-    "moment": "~2.1.0",
-    "i18n": "~0.4.1",
-    "wintersmith": "~2.0.5",
-    "async": "~0.2.9",
-    "slugg": "~0.1.2"
+    "async": "~3.2.0",
+    "i18n": "~0.9.0",
+    "moment": "~2.25.0",
+    "nunjucks": "^3.2.1",
+    "slugg": "~1.2.1",
+    "wintersmith-coffee": "~0.2.2",
+    "wintersmith-stylus": "~0.5.1"
+  },
+  "scripts": {
+    "ws": "wintersmith"
```

Comment:
This was just a way to attach a debugger with my IDE

Reply:
Are you working to revive this project?  It seems @danielgtaylor is still developing on GitHub. I saw something about domain expiration, but that was a few years ago. Would love to see this wonderful resource back up and running.  Happy to help where I can.

Reply:
It started with me wanting to look at the data but ended with hours of debugging; I figured I would share the fix I came up with. Certainly not improving anything yet.

Reply:
Thanks! I'll try and find some time to look into this soon!

Reply:
I managed to open it in Gitpod but can not start adobe flash on page. 
Anybody have any idea why?
I allow flash in webpage....
Bibviz can always be opened from here.
https://web.archive.org/web/20200223132234/http://bibviz.com/
Lets make it work in Gitpod.
Then lets remove flash and use html5 or something else...

Reply:
Adobe killed flash and Google removed it from Chrome.

Reply:
I can open the saved copy from web archive, in Google chrome and see flash graphics, 
when I activate flash in Google chrome.
https://web.archive.org/web/20200223132234/http://bibviz.com/
So it must be possible to open the webpage from the GitHub source code and also see the flash graphic.